### ISSUE TITLE
Storage Drawers

### ISSUE THREAD

**Issue by Enchantica02** [benbenlaw-modpacks/Techopolis-2#25]
Compacting drawers no longer seem to work with compressium cobblestone. Goes into top slot instead of recognizing tiers? Intended or conflict?

*Edit to note all compressium blocks not just cobble

**Comment by benbenlaw** [comment]
yeah im not 100% on how to fix but will try to, its probably to do with gamstages
